consistency-checker/src/main/java/com/redhat/lightblue/migrator/consistency/ConsistencyChecker.java

```diff
@@ -270,6 +283,24 @@ protected static boolean isJobExecutable(MigrationJob job) {
         return executable;
     }
 
+    /**
+     * Update all non-complete job executions as "dead".
+     *
+     * @param job the job to cleanup
+     */
+    protected static void markRunningJobExecutionsAsDead(MigrationJob job) throws IOException {
+        int psn = 0;
+        for (MigrationJobExecution exec : job.getJobExecutions()) {
+            if (exec.getJobStatus() != null && exec.getJobStatus().isRunning()) {
+                LOGGER.info("Marking job {} execution {} as {}", job.get_id(), psn, JobStatus.COMPLETED_DEAD);
```

Comment:
You're asking why check that job status isn't null?  Guess just in case.  I see it's required in metadata so that check could be removed, but why?